src/xrCore/xr_vector_defs.h

```diff
@@ -19,7 +19,7 @@ DEFINE_VECTOR(Fvector, FvectorVec, FvectorIt);
 DEFINE_VECTOR(Fvector*, LPFvectorVec, LPFvectorIt);
 DEFINE_VECTOR(Fcolor, FcolorVec, FcolorIt);
 DEFINE_VECTOR(Fcolor*, LPFcolorVec, LPFcolorIt);
-DEFINE_VECTOR(LPSTR, LPSTRVec, LPSTRIt);
+DEFINE_VECTOR(pstr, LPSTRVec, LPSTRIt);
```

Comment:
Let's leave it as is for now.
```suggestion
DEFINE_VECTOR(LPSTR, LPSTRVec, LPSTRIt);
```